prisma/schema.prisma

```diff
@@ -0,0 +1,137 @@
+datasource db {
+  url      = env("DATABASE_URL")
+  provider = "mysql"
+}
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+model user {
+  id                 Int       @id @default(autoincrement())
+  hashedPassword     String
+  saltPassword       String
+  cluster            Int?
+  isActivated        Boolean   @default(false)
+  isDeleted          Boolean   @default(false)
+  verifyAt           DateTime? @db.DateTime(0)
+  premiumEndsAt      DateTime? @db.DateTime(0)
+  createdAt          DateTime  @default(now()) @db.DateTime(0)
+  isIntroShownFields Json
+  lastOnline         DateTime
+  role               role      @default(USER)
+
+  name        String
+  gender      String
+  status      String
+  orientation String
+  biographic  String?
+
+  email       String  @unique
+  yearOfBirth Int?
+  bodyType    String?
+  diet        String?
+  drinks      String?
+  drugs       String?
+  education   String?
+  ethincity   String?
+  height      Float?
+  income      Float?
+  job         String?
+  location    String?
+  offspring   String?
+  pets        String?
+  sign        String?
+  smokes      String?
+  speaks      String?
+
+  messagesSent     message[]      @relation("messages_sent")
+  messagesReceived message[]      @relation("messages_received")
+  notifications    notification[]
+  coupons          coupon[]
+  userImages       user_image[]
+  reportsMade      report[]       @relation("reports_made")
+  reportsReceived  report[]       @relation("reports_received")
+  likedBy          user[]         @relation("user_likes")
+  liking           user[]         @relation("user_likes")
+  staredBy         user[]         @relation("user_stars")
+  staring          user[]         @relation("user_stars")
+}
+
+model message {
+  id        Int           @id @default(autoincrement())
+  message   String?
+  type      message_types @default(TEXT)
```

Comment:
please move 'type' up 1 line, message should be grouped with imgUrl and voiceUrl